packages/angular/cli/src/commands/cache/long-description.md

```diff
@@ -0,0 +1,47 @@
+Angular CLI saves a number of cachable operations on disk by default.
+
+When you re-run the same build, the build system restores the state of the previous build and re-uses previously performed operations, which decreases the time taken to build and test your applications and libraries.
+
+To amend the default cache settings, add the `cli.cache` object to your [Workspace Configuration](guide/workspace-config).
+The object goes under `cli.cache` at the top level of the file, outside the `projects` sections.
+
+```json
+{
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "version": 1,
+  "cli": {
+    "cache": {
+      ...
```

Comment:
Nit: I believe GitHub supports `jsonc` as a code block language which is read as JSON with comments. That would allow `// ...` to be interpreted as a real comment and avoid the ugly red syntax error.